### ISSUE TITLE
Test Error in Python 3.10

### ISSUE THREAD

**Issue by pravins02** [ubermag/help#248]
Greetings @fangohr 
I tried installing Ubermag on my machine following the instructions from the video and the link shared above and encountered the same error as displayed above. 

<img width="643" alt="Ubermag test" src="https://github.com/ubermag/help/assets/126001166/e26b7d4b-0329-464b-9063-470dac84aa4f">

Please advise me on how to proceed in this aspect.

Thank you

_Originally posted by @pravins02 in https://github.com/ubermag/help/issues/246#issuecomment-1597491220_

**Comment by lang-m** [comment]
Hi @pravins02,
this seems to be a problem with your Anaconda installation on Windows, nothing related to Ubermag.

This stackoverflow thread could maybe help: https://stackoverflow.com/questions/54876404/unable-to-import-sqlite3-using-anaconda-python/55642416#55642416

**Comment by pravins02** [comment]
Thank you for the reply @lang-m but, I uninstalled everything and tried again installing by following the procedures, and its still showing me the same results.
Can you please look into it?

Thank you in advance.

**Comment by fangohr** [comment]
Dear @pravins02, your issue is likely to do with anaconda on your computer, not Ubermag. We cannot reproduce it. @lang-m  has pointed to an stackoverflow issues where somebody else had the same problem, and a solution is reported there. Try to follow that - that's the best advice we can give from here.

You could also try a more recent version of anaconda, try a different user account on your computer, try it on a different computer, etc. to try to figure out what the problem is. We are not very skilled with MS Windows.

**Comment by lang-m** [comment]
Hi @pravins02 based on the screenshot I cannot say why the tests are failing. Could you please share the whole test log with us? You can obtain it by running
```
python -c "import ubermag; ubermag.test()" > ubermag-test-output.txt
```
in the Anaconda Prompt. This will create a new file `ubermag-test-output.txt` (**note**: it would overwrite an existing file with the same name!) that you can upload here.

**Comment by pravins02** [comment]
Hi @lang-m 

As per your suggestion, I am hereby attaching the test log.

[ubermag-test-output.txt](https://github.com/ubermag/help/files/11929799/ubermag-test-output.txt)


Let me know what needs to be done after this to remove this error.

Thank you

**Comment by lang-m** [comment]
Hi @pravins02 

Thank you for sharing the test file.

Three of the errors (`test_read_write_vtk` and both `test_ovf2vtk`) are related to some changes in the dependencies. However, these are only relevant for you, if you need `vtk` export capabilities (which would not be done internally). You can ignore those unless you use and need `field.write("<filename>.vtk")`. If that is the case you can try to install an older version of numpy (`conda install -c conda-forge numpy=1.20`). That should fix these tests.

The last error (`test_dirname`) is related to some permission difficulties when running OOMMF on Windows. This should not affect your actual simulations, only causes the test to fail.

So as a conclusion, the failing tests should not be a major concern for you and you should be able to run simulations with Ubermag without any problems.

To help us resolve the dependency problem related to vtk, could you please upload a full list of your installed packages, which you can obtain by running
```
conda list > ubermag-packages-list.txt
```
in the Anaconda Prompt.

**Comment by pravins02** [comment]
Hi @lang-m, sorry for the delay.

I have tried that and I was getting the Attribute error in the simulation which was earlier discussed in "_AttributeError: Object has no attribute k3d_vector. #87_".

I did follow the solution there and my simulation started to work. 

Thank you for the prompt response and guidance.  However, I am attaching my package list here as suggested by you.

[ubermag-packages-list.txt](https://github.com/ubermag/help/files/11970358/ubermag-packages-list.txt)


Thank you so much
